depends/patches/bzip2/Makefile.patch

```diff
@@ -1,17 +1,21 @@
 --- Makefile	2019-07-13 13:50:05.000000000 -0400
 +++ Makefile	2019-10-28 13:31:36.918640240 -0400
-@@ -15,10 +15,10 @@
+@@ -15,13 +15,13 @@
  SHELL=/bin/sh
-
+ 
  # To assist in cross-compiling
 -CC=gcc
 -AR=ar
 -RANLIB=ranlib
 -LDFLAGS=
-+# CC=gcc
-+# AR=ar
-+# RANLIB=ranlib
-+# LDFLAGS=
-
++#CC=gcc
++#AR=ar
++#RANLIB=ranlib
++#LDFLAGS=
+ 
  BIGFILES=-D_FILE_OFFSET_BITS=64
- CFLAGS=-Wall -Winline -O2 -g $(BIGFILES)
+-CFLAGS=-Wall -Winline -O2 -g $(BIGFILES)
++CFLAGS=-Wall -Winline -O2 -g $(BIGFILES) -fPIC
```

Comment:
The bottom line is that the Makefile ignores the CFLAGS passed in from the package. I put the -fPIC here rather than the CC line in the package, because it is a little cleaner. I am not really happy with the way Bitcoin did this, as it is not immediately obvious that the package level CFLAGS is being tossed.

Reply:
This one is the only change that I needed to get the depends builds to work in a fresh Leap 15.3. We could also inject the options with something like...

```suggestion
+CFLAGS=-Wall -Winline -O2 -g $(BIGFILES) $(DEPENDS_CFLAGS)
```

...and define that in the Makefile. Might be slightly less hacky than pulling the flags into the patch or moving `BIGFILES` into the recipe, but not by much...

Just curious... where did we get the bzip recipe from? It doesn't seem like it's from Bitcoin&mdash;the recipe doesn't look like it exists anywhere in that repo's history.

Reply:
I don't recall actually...

Reply:
On the fresh Leap... I think openSuSE changed their directory structure to conform better to LSB, but there are lots of legacy upgraded openSuSE installations (such as mine) that use a mixed /usr/lib /usr/lib64 directory structure. The other changes in this PR are effectively a no-op if you are using a distribution that uses /usr/lib, but it fixes compilation problems on distros that use lib64.

Reply:
The bzip2 makefile is a hack anyway. Let's leave it alone. It is good enough I think.